tests/test_generate_mbz_toc.py

```diff
@@ -20,10 +22,10 @@ def test_toc_creation_single_page(
     html_path.mkdir()
     mbz_builder(mbz_path, activities=[page])
     mbtools.extract_html_content.replace_content_tags(mbz_path, html_path)
-    md_filepath = tmp_path / "toc.md"
+    md_filepath = tmp_path
```

Comment:
You can probably revert this change you made (which would cascade down to various other small, unnecessary changes in this PR).